src/13_file_io.py

```diff
@@ -10,10 +10,21 @@
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
 # YOUR CODE HERE
+f = open('foo.txt', 'r')
+data = f.read()
+print(data)
+f.close()
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+f = open('bar.txt', 'w')
+f.write("I am at bar.txt")
```

Comment:
Technically should be three lines of text, not one: 

"Write three lines of arbitrary content to that file, then close the file."